### DISCUSSION TITLE
Maintainers call

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/EddieHubLive#86] [Ideas]
Discussion ideas

**Comment by eddiejaoude** [comment]
Page does not appear blank, display the latest X events

**Comment by eddiejaoude** [comment]
Display map on EddieHub website in iframe

**Comment by eddiejaoude** [comment]
Live stream to have the map on half the screen

**Comment by amandamartin-dev** [comment]
long term - ability to connect other projects and repositories outside of eddiehub

**Comment by Pradumnasaraf** [comment]
Clicking of profile picture will redirect user to LinkFree

<img width="891" alt="Screenshot 2022-12-18 at 9 03 02 PM" src="https://user-images.githubusercontent.com/51878265/208306747-abb48487-8455-4d96-a2cb-99d922bd7659.png">
